src/components/productfacets/list/ProductFacetList.tsx

```diff
@@ -0,0 +1,130 @@
+import {Box, Container, Text, useDisclosure} from "@chakra-ui/react"
+import Link from "next/link"
+import {ProductFacets} from "ordercloud-javascript-sdk"
+import {useCallback, useState} from "react"
+import {IProductFacet} from "types/ordercloud/IProductFacet"
+import {DataTableColumn} from "../../shared/DataTable/DataTable"
+import ListView, {ListViewGridOptions, ListViewTableOptions} from "../../shared/ListView/ListView"
+import ProductFacetDeleteModal from "../modals/ProductFacetDeleteModal"
+import ProductFacetActionMenu from "./ProductFacetActionMenu"
+import ProductFacetCard from "./ProductFacetCard"
+import ProductFacetListToolbar from "./ProductFacetListToolbar"
+
+const ProductFacetQueryMap = {
+  s: "Search",
+  sort: "SortBy",
+  p: "Page"
+}
+
+const IDColumn: DataTableColumn<IProductFacet> = {
+  header: "ID",
+  accessor: "ID",
+  width: "20%",
+  cell: ({row, value}) => (
+    <Link href={"/settings/productfacets/" + row.original.ID}>
+      <Text as="a" noOfLines={2} title={value}>
+        {value}
+      </Text>
+    </Link>
+  ),
+  sortable: true
+}
+
+const NameColumn: DataTableColumn<IProductFacet> = {
+  header: "Name",
+  accessor: "Name",
+  width: "20%",
+  cell: ({row, value}) => (
+    <Link href={"/settings/productfacets/" + row.original.ID}>
+      <Text as="a" noOfLines={2} title={value}>
+        {value}
+      </Text>
+    </Link>
+  ),
+  sortable: true
+}
+
+const OptionsColumn: DataTableColumn<IProductFacet> = {
+  header: "Facet Options",
+  accessor: "xp.Options",
+  width: "60%",
+  cell: ({row, value}) => (
+    <Link href={"/settings/productfacets/" + row.original.ID}>
+      <Text as="a" noOfLines={2} title={value}>
+        {value}
+      </Text>
+    </Link>
+  ),
+  sortable: true
```

Comment:
This is not sortable, when attempting to use it in the preview deployment it throws an error.

Reply:
Good catch, code updated.